titus-server-master/src/main/java/com/netflix/titus/master/scheduler/constraint/V3ZoneBalancedFitnessCalculator.java

```diff
@@ -0,0 +1,63 @@
+package com.netflix.titus.master.scheduler.constraint;
+
+import java.util.Map;
+
+import com.netflix.fenzo.TaskRequest;
+import com.netflix.fenzo.TaskTrackerState;
+import com.netflix.fenzo.VMTaskFitnessCalculator;
+import com.netflix.fenzo.VirtualMachineCurrentState;
+import com.netflix.titus.master.jobmanager.service.common.V3QueueableTask;
+import com.netflix.titus.master.scheduler.SchedulerUtils;
+
+public class V3ZoneBalancedFitnessCalculator implements VMTaskFitnessCalculator {
+
+    private final TaskCache taskCache;
+    private final int expectedValues;
+    private final String zoneAttributeName;
+
+    public V3ZoneBalancedFitnessCalculator(TaskCache taskCache, int expectedValues, String zoneAttributeName) {
+        this.taskCache = taskCache;
+        this.expectedValues = expectedValues;
+        this.zoneAttributeName = zoneAttributeName;
+    }
+
+    @Override
+    public String getName() {
+        return "V3ZoneBalancedFitnessCalculator";
```

Comment:
I prefer spaces in between each of the words like the others.

Reply:
I changed other to use camel case like here.